worktree_test.go

```diff
@@ -1488,6 +1488,96 @@ func (s *WorktreeSuite) TestAddRemovedInDirectory(c *C) {
 	c.Assert(file.Staging, Equals, Unmodified)
 }
 
+func (s *WorktreeSuite) TestAddRemovedInDirectoryWithTrailingSlash(c *C) {
```

Comment:
nit: these 2 tests are almost identical, why not do them in a loop, eg:
```
	cases := []struct {
		name       string
		dir        string
		goStatus   string
		jsonStatus string
	}{
	...
	}
```

Reply:
I followed the precedent from the other tests, and didn't want to spend time refactoring them all. If it's fine with you, I'd rather leave it as is.

Reply:
Looks like I can't mark this comment as resolved, it was just a `nit` so all good.